docfx_yaml/extension.py

```diff
@@ -626,6 +628,15 @@ def process_docstring(app, _type, name, obj, options, lines):
     """
     This function takes the docstring and indexes it into memory.
     """
+
+    # Check if we already processed this docstring.
+    if name not in app.env.docfx_uid_names:
+        # Checking in dictionary("{}") takes O(1) average time for a bit more space,
+        # whereas checking in lists("[]") takes O(n) average time.
```

Comment:
Not sure we need this, but it doesn't hurt. Using a dict as a set is standard.

Reply:
done.